tests/beacon/test_helpers.py

```diff
@@ -376,16 +381,60 @@ def mock_get_shards_and_committees_for_slot(parent_block,
 
     if isinstance(result_proposer_index_in_committee, Exception):
         with pytest.raises(ValueError):
-            get_proposer_position(
+            get_block_committees_info(
                 parent_block,
                 genesis_crystallized_state,
                 cycle_length,
             )
     else:
-        proposer_index_in_committee, _ = get_proposer_position(
+        block_committees_info = get_block_committees_info(
             parent_block,
             genesis_crystallized_state,
             cycle_length,
         )
+        assert (
+            block_committees_info.proposer_index_in_committee ==
+            result_proposer_index_in_committee
+        )
+
+
+@pytest.mark.parametrize(
+    (
+        'votes_count'
+    ),
+    [
+        (0),
+        (1),
+        (5),
+        (9),
+    ],
+)
+def test_aggregate_votes(votes_count, privkeys, pubkeys):
+    bit_count = 10
+    pre_bitfield = get_empty_bitfield(bit_count)
+    pre_sigs = ()
+
+    random_votes = random.sample([i for i in range(bit_count)], votes_count)
```

Comment:
We've *typically* tried to use `hypothesis` when we need randomness in tests so that the randomness can be reproduced upon failure.  It's not a hard requirement, but in cases where you are using random inputs to test something and the randomness can be a source of failure, it's probably what you should do.

You can do as far as having hypothesis generate values for you, or if you still just want to use the built-in python `random` standard library module, you can use this https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.random_module